symbology/custom-dictionary-style/src/main/java/com/esri/samples/custom_dictionary_style/CustomDictionaryStyleSample.java

```diff
@@ -0,0 +1,131 @@
+ */
+
+package com.esri.samples.custom_dictionary_style;
+
+import java.util.HashMap;
+import java.util.List;
+
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.layout.StackPane;
+import javafx.stage.Stage;
+
+import com.esri.arcgisruntime.data.ServiceFeatureTable;
+import com.esri.arcgisruntime.layers.FeatureLayer;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.Viewpoint;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.symbology.DictionaryRenderer;
+import com.esri.arcgisruntime.symbology.DictionarySymbolStyle;
+import com.esri.arcgisruntime.symbology.DictionarySymbolStyleConfiguration;
+
+public class CustomDictionaryStyleSample extends Application {
+
+  private MapView mapView;
+
+  @Override
+  public void start(Stage stage) {
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene scene = new Scene(stackPane);
+
+      // create a map view
+      mapView = new MapView();
+
+      // set title, size, and add scene to stage
+      stage.setTitle("Custom Dictionary Style Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // open the custom style file
+      DictionarySymbolStyle restaurantStyle = DictionarySymbolStyle.createFromFile("./samples-data/stylx/Restaurant.stylx");
+
+      // create a new map with a streets basemap and show it
+      ArcGISMap map = new ArcGISMap(Basemap.createStreetsVector());
+      mapView.setMap(map);
+
+      // create the restaurants feature table from the feature service
+      ServiceFeatureTable restaurantFeatureTable = new ServiceFeatureTable("https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/arcgis/rest/services/Redlands_Restaurants/FeatureServer/0");
+
+      // create the restaurants layer and add it to the map
+      FeatureLayer restaurantLayer = new FeatureLayer(restaurantFeatureTable);
+      map.getOperationalLayers().add(restaurantLayer);
+
+      // set the map's initial extent to that of the restaurants layer
+      restaurantLayer.loadAsync();
+      restaurantLayer.addDoneLoadingListener(() ->
+              map.setInitialViewpoint(new Viewpoint((restaurantLayer.getFullExtent()))));
+
+      // create overrides for expected field names that are different in this dataset
+      HashMap<String, String> styleToFieldMappingOverrides = new HashMap<>();
+      styleToFieldMappingOverrides.put("style", "Style");
+      styleToFieldMappingOverrides.put("price", "Price");
+      styleToFieldMappingOverrides.put("healthgrade", "Inspection");
+      styleToFieldMappingOverrides.put("rating", "Rating");
+
+      // create overrides for expected text field names (if any)
+      HashMap<String, String> textFieldOverrides = new HashMap<>();
+      textFieldOverrides.put("name", "Name");
```

Comment:
this also apparently doesn't change anything: however, it is required for the constructor on line 97. It would be worth making a comment that in this instance there is nothing to change.